routing/notifications.go

```diff
@@ -318,11 +319,19 @@ func addToTopologyChange(graph *channeldb.ChannelGraph, update *TopologyChange,
 		if err != nil {
 			return err
 		}
+
+		globalFeaturesBuffer := new(bytes.Buffer)
+		err = m.Features.Encode(globalFeaturesBuffer)
+		if err != nil {
+			return err
```

Comment:
Is it worth logging an error in this case?

e.g. `log.Errorf("unable to encode features: %v", err)`

Reply:
Imo logging from the caller is enough in this case.